src/easier68k/core/opcodes/move.py

```diff
@@ -1,12 +1,12 @@
 """
->>> str(Move.from_str('MOVE.B', '-(A0), D1')[0])
```

Comment:
Now that this method isn't in a class, we can move these doctests back into the actual method instead of the top of the file.

Reply:
not sure if placement in the file is a big deal currently

Reply:
It's not, just throwing it out there for file cleanliness at some point.